### DISCUSSION TITLE
Embedding of Gemini Google

### DISCUSSION THREAD

**Discussion by jamesdias** [tryAGI/LangChain#369] [Ideas]
There is a plan for the embedding of Gemini Google to be made available?

**Comment by HavenDV** [comment]
Yes, I can create an issue for this, but progress mostly depends on the help of the community, I can help with the initial implementation and make a template.
Is there any C# code that I can rely on?

**Reply by jamesdias** [reply]
Sorry, could you explain to me better what you need in relation to this section?

"Is there any C# code I can trust?"
